src/Uno.UI.FluentTheme.v2/themeresources_v2.xaml

```diff
@@ -10004,11 +10004,8 @@
                   <Storyboard>
                     <!-- Uno specific: avoid applying LinearGradientBrush -->
                     <!--<ObjectAnimationUsingKeyFrames Storyboard.TargetName="BorderElement" Storyboard.TargetProperty="BorderBrush">
-                                            <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource ControlStrokeColorDefaultBrush}" />
+                                            <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource TextControlBorderBrushPointerOver}" />
```

Comment:
Is this part of this change?

Reply:
Yes, it was a forgotten merge, so I included it here.